app/searchcriteria.py

```diff
@@ -42,7 +42,7 @@ async def store_search_criteria(request, search_criteria, fields_to_load=[]):
       session[atribute] = search_criteria.get(atribute)
 
 
-async def clear_stored_search_criteria(session, microservice_name=''):
+async def clear_stored_search_criteria(session, request, microservice_name=''):
```

Comment:
If we are passing `request`, we do not need to pass `session`

Reply:
Just pull the session out at the start of the method.

Reply:
ADjusted to get session from r equest